### ISSUE TITLE
Retrieve user profile info upon login

### ISSUE THREAD

**Issue by itsjoekent** [DoSomethingArchive/legacy-website#6995]
When a user logs in via Facebook we need to grab basic profile information from them,
- [x] First name
- [x] Last name
- [x] Email

Other details _could_ be grabbed, but I would rather not block us from continuing by waiting on the Facebook review process. If we pass review at some point we'll just make another issue / reopen this to get the rest of the profile data.

**Comment by itsjoekent** [comment]
checking all of these off, was addressed in this commit: https://github.com/deadlybutter/phoenix/commit/6bf1a8e0b4569e033bccb366d38403853d232063
